src/misc.js

```diff
@@ -73,48 +73,48 @@ const makeLeader = function(name, cost, rewards, tooltip) {
   return new Card(name, 0, [cost], CardType.LEADER, rewards, 0, tooltip);
 }
 
-const scienceScore = function(scores) {
-  return scores[0] * scores[0] + scores[1] * scores[1] + scores[2] * scores[2] + 7 * Math.min.apply(null, scores);
+const scienceScore = function(player, scores) {
```

Comment:
Instead of passing in the entire player, pass just the scienceSetBonus.

Reply:
Done